Symfony/CS/Tests/Fixer/PSR2/MultipleUseFixerTest.php

```diff
@@ -40,21 +40,22 @@ public function provideCases()
             ),
             array(
                 <<<'EOF'
-use Some, Not, PHP, Like, Use, Statement;
```

Comment:
(btw, my thought while removing the line: hmm, invalid code, is this line necessary? we are using tokens, and do not test php-like non-code text in other fixer tests. So just remove this line, which is probably a relict of regex era of this project.)